### PR TITLE
Update README.adoc

### LINKED ISSUE #44: ID Review
Review for basic ID concerns (grammar, style etc) as well as overall cohesiveness and clarity of content.

### PR DESCRIPTION
**Pull request by priyyanshi**
Addressing #44

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -47,28 +47,29 @@ If you're dealing with large amounts of data or large resources,
...
+GraphQL calculates these properties only if they are requested. 
 
-A GraphQL service can also be used to obtain data from multiple sources such as APIs, databases, or other services.
-It can then collate this data into a single object for the user.
-This simplifies data retrieval as the user only makes a single request to the GraphQL service,
-instead of multiple requests to the individual data sources.
+A GraphQL service can also be used to obtain data from multiple sources such as APIs, databases, and other services. 
+It can then collate this data into a single object for the user, simplifying the data retrieval. 
+The user makes only a single request to the GraphQL service, instead of multiple requests to the individual data sources.
+Compared to REST, GraphQL services require less data fetching resulting in lower applications loading times and lower data transfer costs,
```
Comment (dmuelle) [inline README.adoc]: suggested edit:

GraphQL services require less data fetching than REST services, which results in lower application load times and lower data transfer costs. GraphQL also enables clients to better customize requests to the server.

File: README.adoc [hunk 2]
```diff
@@ -93,17 +94,18 @@ include::{common-includes}/gitclone.adoc[]
 
 Navigate to the `start` directory to begin.
 
-Object types define the structure of the data returned by GraphQL. 
-Annotations that are applied to the declaration and properties of Java classes define object types.
-The object types are defined in their own Maven module that is included in the other modules.
+Object types determine the structure of the data that GraphQL returns. 
+These object types are defined by annotations that are applied to the declaration and properties of Java classes. 
+
+You will define object types `java`, `systemMetrics`, and `systemInfo` by creating and applying annotations to the `JavaInfo`, `SystemMetrics`, and `SystemInfo` classes respectively.
```
Comment (dmuelle) [inline README.adoc]: You will define  `java`, `systemMetrics`, and `systemInfo` object types..

**Review by dmuelle — CHANGES_REQUESTED** [review]
looks good, just two minor edit suggestions

**Review by dmuelle — APPROVED** [review]
LGTM, thanks

**Review by ramkumar-k-9286 — APPROVED** [review]

**Comment by ramkumar-k-9286** [comment]
LGTM

Regards,
Ramkumar

**Pull request merged**